.github/workflows/on_push.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy new tag
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Do we also want to add master to the list?

Reply:
This is just for this repository, and we have `main` branch as the active one. So no need to have `master` in here.